static/js/settings_org.js

```diff
@@ -772,6 +782,13 @@ exports.build_page = function () {
         } else if (subsection === "other_settings") {
             const code_block_language_value = exports.default_code_language_widget.value();
             data.default_code_block_language = JSON.stringify(code_block_language_value);
+            const oss_showcase_policy = $("#id_realm_oss_showcase_policy").val()
+
+            if (oss_showcase_policy === "listed") {
+                data.realm_oss_showcase_policy = 1;
+            } else if (oss_showcase_policy === "hidden") {
+                data.realm_oss_showcase_policy = 2;
+            }
         } else if (subsection === "other_permissions") {
```

Comment:
The select element, in this case, is in the subsection `org_profile` so you might need to add a separate else if block here. And also, it should be `data.oss_showcase_policy`. 
(Side note - These changes are not required in case you decide to keep option values as integers.)